src/test/java/com/wikia/webdriver/testcases/discussions/SharingTests.java

```diff
@@ -29,25 +34,46 @@
   @Test(groups = "discussions-anonUserOnMobileCanSeeSocialNetworkIcons")
   @Execute(asUser = User.ANONYMOUS)
   @InBrowser(browserSize = MOBILE_RESOLUTION)
-  public void anonUserOnMobileCanSeeSocialNetworkIcons() {
-    toggleShareIconClickDisplaysSocialNetworkIcons(expected_networks_for_english_language);
+  public void anonUserOnMobileCanSeeSocialNetworkIconsOnPostListPage() {
+    List<String> socialNetworkNames = findSocialNetworksNamesForFirstPostOnPostListPage();
+
+    Assertion.assertEquals(socialNetworkNames, EXPECTED_SOCIAL_NETWORKS_FOR_ENGLISH_LANGUAGE,
+        "Displayed social networks are different than expected.");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Displayed social networks are different than expected." 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)